cookbook/email/spool.rst

```diff
@@ -55,10 +55,15 @@ swiftmailer with the memory option, use the following configuration:
             'spool' => array('type' => 'memory')
         ));
 
-Spool Using a File
+Spool Using Files
```

Comment:
With a little icon next to each headline on symfony.com, we allow the readers to link to specific sections on a page. This is done by using the automatically created anchor (slugified headline text).

If we change the headline, the automatically generated anchor changes as well (in this case it was `spool-using-a-file` and now is `spool-using-files`). This means that all links to the specific section are broken. If we add the old anchor as well, both anchors will link to this section. It's like backwards compatibility for documentations.